katdal/visdatav4.py

```diff
@@ -185,12 +229,14 @@ def _before(date):
         self.dump_period = attrs['int_time']
         # The CBF dump period is not in the lite RDB version
         try:
-            src_stream = attrs['src_streams'][0]
+            correlator_stream = attrs['src_streams'][0]
             # XXX: should use telstate.join if attrs is a telstate
-            self.cbf_dump_period = attrs[src_stream + '_int_time']
-            cbf_n_accs = attrs[src_stream + '_n_accs']
+            self.cbf_dump_period = attrs[correlator_stream + '_int_time']
+            cbf_n_accs = attrs[correlator_stream + '_n_accs']
+            self._cbf_instrument = attrs[correlator_stream + '_instrument_dev_name']
         except (KeyError, IndexError):
             self.cbf_dump_period = self.accumulations_per_dump = None
+            self._cbf_instrument = 'UNKNOWN'
```

Comment:
Maybe None would be better (as you'd done for dump information).

Reply:
I'm removing the need for this.